Sources/StreamChatUI/ChatChannel/ChatMessage/ChatMessageGroupPart.swift

```diff
@@ -4,15 +4,17 @@
 
 import StreamChat
 
+public typealias ChatMessageGroupPart = _ChatMessageGroupPart<NoExtraData>
+
 @dynamicMemberLookup
 public struct _ChatMessageGroupPart<ExtraData: ExtraDataTypes> {
-    public let message: _ChatMessage<ExtraData>
-    public let quotedMessage: _ChatMessage<ExtraData>?
-    public let isLastInGroup: Bool
-    public let didTapOnAttachment: ((_ChatMessageAttachment<ExtraData>) -> Void)?
-    public let didTapOnAttachmentAction: ((_ChatMessageAttachment<ExtraData>, AttachmentAction) -> Void)?
+    let message: _ChatMessage<ExtraData>
+    let quotedMessage: _ChatMessage<ExtraData>?
+    let isLastInGroup: Bool
+    let didTapOnAttachment: ((_ChatMessageAttachment<ExtraData>) -> Void)?
+    let didTapOnAttachmentAction: ((_ChatMessageAttachment<ExtraData>, AttachmentAction) -> Void)?
 
-    public var isPartOfThread: Bool {
+    var isPartOfThread: Bool {
```

Comment:
Why did you remove the `public` here?

Reply:
Honestly, that's a surprise. I didn't manually change these access modifiers

Reply:
fixed in 6e3e69bbc8ecb5eed62ba790dd05d226cf64444d